themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties

```diff
@@ -0,0 +1,48 @@
+emailTestBody=Toto je testovací zpráva
+emailTestBodyHtml=<p>Toto je testovací zpráva </p>
+identityProviderLinkSubject=Odkaz {0}
+identityProviderLinkBody=Někdo chce propojit váš účet "{1}" s účtem {0}"uživatele {2} . Pokud jste to vy, klikněte na níže uvedený odkaz a propojte účty. \n\n{3}\n\nTento odkaz vyprší za {4} minuty.\n\nPokud nechcete propojit účet, ignorujte tuto zprávu . Pokud propojíte účty, budete se moci přihlásit do {1} až {0}.
+identityProviderLinkBodyHtml=<p> Někdo chce propojit váš účet <b>{1}</b> s účtem <b>{0}</b> uživatele {2} . Pokud jste to vy, kliknutím na odkaz níže propojíte účty </p><p><a href="{3}">Odkaz na potvrzení propojení</a></p><p>Tento odkaz vyprší v rámci {4} minut.</p><p>Pokud nechcete propojit účet, ignorujte tuto zprávu. Pokud propojíte účty, budete se moci přihlásit do {1} až {0}.</p>
+passwordResetSubject=Zrušit heslo
+passwordResetBody=Někdo právě požádal o změnu pověření účtu {2}. Pokud jste to vy, klikněte na odkaz níže, abyste je resetovali.\n\n{0}\n\nTento odkaz a kód uplynou do {1} minut.\n\nPokud nechcete obnovit vaše pověření , ignorujte tuto zprávu a nic se nezmění.
+passwordResetBodyHtml=<p> Někdo právě požádal o změnu pověření vašeho účtu {2}. Pokud jste to vy, klikněte na odkaz níže, abyste je resetovali.</p><p><a href="{0}">Odkaz na obnovení pověření </a></p><p> Platnost tohoto odkazu vyprší během {1} minut.</p><p> Pokud nechcete obnovit vaše pověření, ignorujte tuto zprávu a nic se nezmění.</p>
+executeActionsSubject=Aktualizujte svůj účet
+executeActionsBody=Váš administrátor právě požádal o aktualizaci svého účtu {2} provedením následujících akcí: {3}. Kliknutím na níže uvedený odkaz začnete tento proces.\n\n{0}\n\nTento odkaz vyprší za {1} minuty.\n\nPokud si nepřejete, že to váš správce požádal, ignorujte tuto zprávu a nic budou změněny.
+executeActionsBodyHtml=<p> Správce právě požádal o aktualizaci svého účtu {2} provedením následujících akcí: {3}. Kliknutím na odkaz uvedený níže spustíte tento proces.</p><p><a href="{0}">Odkaz na aktualizaci účtu </a></p><p> Tento odkaz vyprší v období {1} minuty.</p><p> Pokud si nejste jisti, že váš správce o to požádal, ignorujte tuto zprávu a nic se nezmění.</p>
+eventLoginErrorSubject=Chyba přihlášení
+eventLoginErrorBody=Do účtu {0} z {1} byl nalezen neúspěšný pokus o přihlášení. Pokud jste to nebyli vy, kontaktujte administrátora.
+eventLoginErrorBodyHtml=<p> Do účtu {0} z {1} byl nalezen neúspěšný pokus o přihlášení. Pokud jste to nebyli vy, kontaktujte správce.</p>
+eventRemoveTotpSubject=Odebrat TOTP
+eventRemoveTotpBody=TOTP byl ze svého účtu odebrán z {0} z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.
+eventRemoveTotpBodyHtml=<p> TOTP byl ze svého účtu odebrán z {0} z {1}. Pokud jste to nebyli vy, kontaktujte správce.</p>
+eventUpdatePasswordSubject=Aktualizovat heslo
+eventUpdatePasswordBody=Vaše heslo bylo změněno z {0} z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.
```

Comment:
V účtu {0} bylo změněno heslo z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.